resources/public/api/conf/1.0/application.yaml

```diff
@@ -55,195 +58,7 @@ paths:
-        - $ref: '#/components/parameters/authorisationParam'
-        - $ref: '#/components/parameters/xFwdHostParam'
-        - $ref: '#/components/parameters/dateParam'
-        - $ref: '#/components/parameters/xClientIdParam'
-        - $ref: '#/components/parameters/eoriParam'
-        - $ref: '#/components/parameters/recordIdParam'
-        - $ref: '#/components/parameters/updatedSinceParam'
-      tags:
-        - Goods records
-      summary: Get good records by EORI and recordId
-      description: Helps read Good records by EORI and recordId
-      responses:
-        '200':
-          description: OK;   Request completed successfully
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/UpdateTGPRecordResponseSchema'
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/xCorrelationIdHdr'
-        '400':
-          description: Bad Request; Incorrect request parameters provided.
-        '401':
-          description: Unauthorised; Credentials failed
-        '403':
-          description: Forbidden;   User was valid, but not allowed to perform this action
-        '404':
-          description: TGP data not found for the search criteria
-      security:
-        - userRestricted:
-            - trader-goods-profiles
-  /customs/traders/goods-profiles/{eori}:
-    put:
-      parameters:
-        - $ref: '#/components/parameters/authorisationParam'
-        - $ref: '#/components/parameters/xFwdHostParam'
-        - $ref: '#/components/parameters/dateParam'
-        - $ref: '#/components/parameters/xClientIdParam'
-        - $ref: '#/components/parameters/eoriParam'
-      tags:
-        - Traders profile
-      summary: Maintain a trader's profile record
-      description: Maintain a trader's profile record
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/MaintainProfileRecordSchema'
-        required: true
-      responses:
-        '200':
-          description: OK; Request completed successfully
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/MaintainProfileRecordResponse'
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/xCorrelationIdHdr'
-        '400':
-          description: Bad Request; Incorrect request parameters provided.
-        '401':
-          description: Unauthorised; Credentials failed
-        '403':
-          description: Forbidden;   User was valid, but not allowed to perform this action
-        '404':
-          description: Maintain profile records not found to deactivate
-      security:
-        - userRestricted:
-            - trader-goods-profiles
+            - write:trader-goods-profiles
 components:
   schemas:
     CreateTGPRecordSchema:
```

Comment:
The other schemas (e.g. Update) are still here and should probably be removed, I'm not sure if they would appear in the devhub or not

Reply:
Ah ok, let me remove them just to be safe